src/classes/AFFL_MultiRecordType_TDTM.cls

```diff
@@ -74,8 +74,18 @@ public class AFFL_MultiRecordType_TDTM extends TDTM_Runnable {
 				for (SObject so : newlist) {
 					Affiliation__c affl = (Affiliation__c)so;
 
-					String lookupFieldLabel = afflMapper.mapAccRecTypeToContactLabel.get(affl.Affiliation_Type__c);
-					String lookupFieldName = afflMapper.contactLabelNames.get(lookupFieldLabel);
+					String lookupLabelOrAPINameFromMapping = afflMapper.mapAccRecTypeToContactLabel.get(affl.Affiliation_Type__c);
+					String lookupFieldName = afflMapper.contactLabelNames.get(lookupLabelOrAPINameFromMapping);
+          
+					// If lookupFieldName of the Contact is not found, check if lookupLabelOrAPINameFromMapping is an API, if yes assign it to lookupFieldName
+					// This code block below is to make sure that Primay Affiliation lookup is populated on contact even when language is not English
```

Comment:
We don't really need to say "This code block below is to" in the commend. It should be understood that the comment refers to the code below. Also, Can we make sure the comments in lines 80 and 81 work together to provide context? Thanks!